lang/README.md

```diff
@@ -6,4 +6,32 @@ SPDX-License-Identifier: MIT
 
 # What is this directory?
 
-This directory and it's contents are a placeholder for the future git submodule that will manage translation files for both this repository and the main game repository, that will it self be managed through transifax.
+This directory and its contents are a placeholder for the future git submodule that will manage translation files for both this repository and the main game repository, which will itself be managed through Transifex.
+
+### Do not edit any files in the `lang` folder manually, except for this READEME.md
+
+All other files are managed automatically!
```

Comment:
No, they are not. They *will* be in the future, as written in the paragraph that exits at the top and tracked in https://github.com/beyond-all-reason/bar-lobby/issues/424

And once this directory will be a submodule, this readme file won't even exist, so this text won't exist.

The current process is described in https://github.com/beyond-all-reason/bar-lobby/wiki/Internationalization and it says "To create a new translation string, you want to first edit the ./lang/en/lobby.json to include the new strings"